int32 estimated_altitude
int16 variation
